src/css/header-menu.css

```diff
@@ -0,0 +1,157 @@
+.header-menu-section {
+    position: fixed;
+
+    width: 100%;
+    height: 100%;
+    z-index: 999;
+    top: 0;
+    left: 0;
+    display: flex;
+    flex-direction: column;
+
+    background-color: #0e1823;
+
+    opacity: 0;
+    pointer-events: none;
+    visibility: hidden;
+    overflow: auto;
+
+    transition: opacity 250ms cubic-bezier(0.4, 0, 0.2, 1), visibility 250ms cubic-bezier(0.4, 0, 0.2, 1);
+
+}
+
+.is-open {
+    opacity: 1;
+    pointer-events: all;
+    visibility: visible;
+    overflow: auto;
+}
+
+.menu-section-plate {
+    background-color: #fbfbfb;
+
+    display: flex;
+    align-items: center;
+    border: 1px solid #fbfbfb;
+    border-radius: 8px;
+    padding-right: 20px;
+
+    margin-right: 20px;
+    margin-left: 20px;
+    margin-top: 16px;
+}
+
+.text-menu-section {
+    font-family: "Inter", sans-serif;
+    font-weight: 500;
+    font-size: 14px;
+    letter-spacing: -0.03em;
+    color: #1b414b;
+
+    padding-left: 20px;
+}
+
+.text-menu-section:active,
+.text-menu-section:hover {
+    color: #055545;
+}
+
+.btn-close-header-menu {
+    height: 28px;
+    width: 28px;
```

Comment:
svg has to be 28x28 in html markup as an attributes, not the button. Button maybe too, but you will see.